Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +199,19 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        tab["HEAD"] = formula_last_commit_hash
+        tab["time"] = formula_last_commit_time.to_i
+        tab.write
```

Comment:
It's debatable for me whether `HEAD` or `time` are useful in the tab file at all; I've never seen them used in debugging and they aren't used anywhere in the code. If we really want to keep these attributes as-is they are we need to create a separate tab file that lives outside the bottle (and therefore also possible the Cellar) otherwise you cannot make a reproducible build if you have your Homebrew core commit at a different version (which is an unnecessary limitation). Another option would be not installing the tab files in bottles or not just not including `HEAD` or `time` in bottles.